io.mrarm.mcpelauncher-ui.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "io.mrarm.mcpelauncher-ui",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.14",
+    "command": "mcpelauncher-ui-qt",
+    "build-options" : {
+        "cflags": "-O2 -g -DNDEBUG",
+        "cxxflags": "-O2 -g -DNDEBUG"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/bin/protoc",
+        "/bin/zipcmp",
+        "/bin/zipmerge",
+        "/bin/ziptool",
+        "/lib/libprotobuf-lite*",
+        "/lib/libprotoc*"
+    ],
+    "finish-args": [
+        "--socket=x11",
+        "--share=network",
+        "--filesystem=host",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Flatpak"
+     ],
+    "modules": [
+        {
+            "name": "protobuf",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/protocolbuffers/protobuf.git",
+                    "branch": "3.12.x"
+                }
+            ]
+        },
+        {
+            "name": "libzip",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/nih-at/libzip.git",
+                    "branch": "master"
+                }
+            ],
+            "buildsystem": "cmake-ninja"
+        },
+        {
+            "name": "libuv",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/libuv/libuv.git",
+                    "branch": "v1.x"
+                }
+            ],
+            "buildsystem": "cmake-ninja"
+        },
+        {
+            "name": "mcpelauncher-ui",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/minecraft-linux/mcpelauncher-ui-manifest.git",
+                    "branch": "ng"
+                }
+            ],
+            "buildsystem": "cmake-ninja"
+        },
+        {
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/minecraft-linux/mcpelauncher-proprietary.git",
+                    "branch": "master"
+                },
+                {
+                    "type": "file",
+                    "path": "io.mrarm.mcpelauncher-ui.metainfo.xml"
+                }
+            ],
+            "name": "compat",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/bin/",
+                "echo '#!/bin/bash' > /app/bin/mcpelauncher-client",
+                "echo 'flatpak-spawn --host flatpak --filesystem=~/.var/app/$FLATPAK_ID:ro run app/io.mrarm.mcpelauncher-client// \"$@\"' >> /app/bin/mcpelauncher-client",
```

Comment:
As I said in separate comment, `flatpak-spawn --host flatpak` is a no-go.

Reply:
Removed